octobot_backtesting/data/database.py

```diff
@@ -21,6 +21,7 @@
 
 import octobot_backtesting.enums as enums
 import octobot_backtesting.errors as errors
+import octobot_backtesting.data.cursor_pool as cursor_pool
 
 
 class DataBase:
```

Comment:
I think we'll have to migrate this class to use Commons database classes

Reply:
You mean using a json db instead of sqlite or move this sqlite database to commons ?